src/main/java/io/snowdrop/github/reporting/GithubReportingService.java

```diff
@@ -1,19 +1,92 @@
 package io.snowdrop.github.reporting;
 
+import java.util.Collection;
+import java.util.stream.Collectors;
+
+import javax.enterprise.context.control.ActivateRequestContext;
+import javax.enterprise.event.Observes;
 import javax.inject.Inject;
 import javax.inject.Singleton;
+import javax.transaction.Transactional;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
+import io.quarkus.runtime.StartupEvent;
 import io.quarkus.scheduler.Scheduled;
+import io.snowdrop.github.reporting.model.IssueDTO;
+import io.snowdrop.github.reporting.model.PullRequestDTO;
+import io.snowdrop.github.reporting.model.RepositoryDTO;
 
 @Singleton
 public class GithubReportingService {
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(GithubReportingService.class);
+
   @Inject
   GithubReporting reporting;
 
+  @ActivateRequestContext
```

Comment:
Remind please why this is needed?

Reply:
I was having issues when trying to do jpa stuff from `onStart`. 
After searching for a while, I bumped to post suggesting adding the annotation.

Reply:
OK, let's keep it